queue/abstract/driver/utube.lua

```diff
@@ -93,14 +93,12 @@ function method.delete(self, id)
     self.space:delete(id)
     if task ~= nil then
         task = task:transform(2, 1, state.DONE)
+        self.on_task_change(task, 'delete')
```

Comment:
What is the purpose of the on_task_change() moving?

Reply:
I don't remember. You can put it back below the `local neighbor` declaration to be safe.

PS: I'm not using tarantool atm, so you'll have to wrap this PR up on your own. ;)

Reply:
Ok